caom2utils/caom2utils/caom2blueprint.py

```diff
@@ -1835,6 +1835,19 @@ def configure_time_axis(self, axis, override=True):
 
         self._time_axis_configed = True
 
+    def set(self, caom2_element, value, extension=0):
+        """
+        Sets the value associated with an element in the CAOM2 model. Value
+        cannot be a tuple.
+        :param caom2_element: name CAOM2 element (as in
+        ObsBlueprint.CAOM2_ELEMEMTS)
+        :param value: new value of the CAOM2 element
+        :param extension: extension number (used only for Chunk elements)
+        """
+        if isinstance(value, numpy.bytes_):
+            value = value.decode('utf-8')
```

Comment:
This override is just to handle the numpy case? Could the caller do the conversion instead of adding this new method? Or can this method handle a more general case where value is bytes? (don't know the relationship between numpy.bytes_ and python bytes)

Reply:
The caller references values retrieved by the `BlueprintParser._get_from_list` method, so making efficient use of 
 that return value would just result in moving the conversion location in this module.

I changed the implementation to `hasattr(value, 'encode')`. I'm not sure if that's the effect you were looking for?

Reply:
That seems to be more generic.